baseline-status.js

```diff
@@ -290,7 +302,39 @@ export class BaselineStatus extends LitElement {
       return `Baseline: ${title}${year ? ` ${year}` : ''}${badge ? ` (newly available)` : ''}. Supported in Chrome: ${chrome === 'available' ? 'yes' : chrome}. Supported in Edge: ${edge === 'available' ? 'yes' : edge}. Supported in Firefox: ${firefox === 'available' ? 'yes' : firefox}. Supported in Safari: ${safari === 'available' ? 'yes' : safari}.`;
     };
 
-    return html` <h1>${feature.name}</h1>
```

Comment:
Any reason to not do the following?:
`<div class="baseline-status-name">${feature.name}</div>`

Then we could change the css above to:

```diff
-      h1,h2,h3,h4,h5,h6 {
+      div.baseline-status-name {
        font-weight: normal;
        font-size: 20px;
        margin: 0;
      }
```

And you could get rid of the switch statement.

You will get the same styling and the component does not have to expose another argument.

Reply:
That certainly would simplify things. The issue then is that we lose a heading in the component for accessibility. 

An alternate would be using `<div role="header">`, although some users may still wish to set the `aria-level` (which defaults to 2). 

If an incorrect heading level impacts the accessibility, I think we would still want to expose it as an option.

Reply:
Since it is a widget and not an element intended to be used exclusively in prose, I'd rather omit the <h1> header element and stick to simple divs. E.g. if the widget is used in a sidebar, h1..h6 headings make little sense, even if their level is configurable. The user can add a heading to the section containing the widget if they wish to use it in prose outside of the widget.

Reply:
Was asked by @jamesnw to chip in here. [The original issue](https://github.com/web-platform-dx/baseline-status/issues/28) was around the risk of repetitive use of `<h1>` on the page, so naturally `<h2>` and lower have been offered here. However as @devnook said, there's little semantic value to add here at all. Here's my two suggestions:
1. Use a `<div>` like @jcscottiii and @devnook suggested, but wrap it with a `<slot name="header>`  so people can apply their own heading level. They'll have to repeat the title, but that's the sacrifice they have to make. Or…
2. Still use the `<slot name="header">` but wrap it around a default `<h2>`, and then somewhere (host or an immediate child) add a `<section>` or `role="section"` to dive the heading semantic meaning.

I'm not precious about which way to go, just would like the `<h1>` to be remove for the same reason Nathan mentioned in his issue. I offered the slot option since I've used it for people in the past and it's a nice escape hatch for users without ending up with complex checks like seen here.

Reply:
> 1. Use a `<div>` like @jcscottiii and @devnook suggested, but wrap it with a `<slot name="header>`  so people can apply their own heading level. They'll have to repeat the title, but that's the sacrifice they have to make. Or…

I think providing a slot makes sense, although it's tricky as the title is provided by the component (the `id` is provided by the user which loads the data, including the title). The ways I can think of having the consumer provide a template and the component provide the content are fairly complicated.

My current thinking is that since `h1` is definitely wrong, and there's little semantic meaning to be provided here, and since we don't have an easy way to make the content available in a slotted content, we should just have it be a `div`. And then, as people use it, make sure we track accessibility and adjust as needed.